src/gennylib/include/gennylib/Orchestrator.hpp

```diff
@@ -104,6 +103,10 @@ class Orchestrator {
      */
     bool continueRunning() const;
 
+    /**
+     * Sleep until the given phase ends or time out.
```

Comment:
What if the timeout occurs? What is the expected behavior for the rate-limiter? I don't know what the right answer is, but the possibility for timeouts seems real esp if we have very long phases and a very tight rate limits.

Reply:
Timeouts here are actually expected! The rate-limiter wraps its sleeps in a loop that checks whether it can consume a token, sleeping for the rate if not. The doc comment was bad it should really say "sleep for the duration, also waking up if the phase changes before the duration passes". I'll update to clarify.